_data/banking.yml

```diff
@@ -556,6 +556,14 @@ websites:
     tfa:
       - sms
 
+  - name: Fineco Bank
+    url: https://finecobank.com/
+    img: finecobank.png
+    tfa:
+      - sms
+      - proprietary
+    doc: https://finecobank.com/it/online/sicurezza/
+
```

Comment:
```suggestion
    regions:
      - it

```

Reply:
Done — you may want to update the docs tho https://github.com/2factorauth/twofactorauth/wiki/FAQ-New-2FA-Supported-Site

Reply:
Actually I'll add UK too, since it's both IT and UK

Reply:
Done that, too https://github.com/2factorauth/twofactorauth/pull/4882/commits/46dcec75a777f72f5f3deef4a8ae146231db6ebc